docs/style-guides/js.md

```diff
@@ -0,0 +1,252 @@
+# origin-web-console JS style guide
+
+A reasonable approach to assisting in style consistency in `openshift-web-console`
+inspired by guides such as the style guide published by [airbnb](https://github.com/airbnb/javascript).
+
+## ESLint
+
+Run `grunt eslint` to check code for errors and style discrepancies.
+
+## Examples
+
+
+### Variables
+
+Group your variable names but use individual `var` statements:
```

Comment:
I actually disagree with this one on multiple fronts. If you are just declaring a bunch of vars and not assigning them, I prefer a single var

```
var foo, bar, baz;
```
Variables that are assigned values should get their own var and their own line.

The second point I disagree with is grouping them. I think it really depends on the context.